matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -1,7 +1,7 @@
 Library {
   Name			  "WBToolboxLibrary_repository"
   Version		  8.4
-  SavedCharacterEncoding  "UTF-8"
+  SavedCharacterEncoding  "windows-1252"
```

Comment:
This seems dangerous.

Reply:
Yep. One thing I remember I was doing is, when possible, picking up only the chunks of the mdl that were directly related to the newly added / modified block (quite difficult from the command line, surprisingly easy from tools like GitKraken). I'm not sure if this could be done every time due to the re-numbering of the IDs.

Reply:
This was fixed by calling `slCharacterEncoding('UTF-8')` in MATLAB. In general while editing the library running: 
~~~
set_param(gcs,'EnableLBRepository','on');
slCharacterEncoding('UTF-8')
~~~
is beneficial.

Reply:
It can definitely be added in the export script